ipywidgets/static/widgets/js/widget.js

```diff
@@ -548,7 +563,11 @@ define(["./utils",
         }
     };
 
-    var DOMWidgetModel = WidgetModel.extend({}, {
+    var DOMWidgetModel = WidgetModel.extend({
+        defaults: _.extend({}, WidgetModel.prototype.defaults, {
+            layout: undefined,
```

Comment:
Well I don't know. We don't allow `None` for the layout widget. 
In fact, even when building it from JavaScript, we should instantiate a layout widget. Not sure what is the best way to do it. That is why I used `undefined`.